packages/driver/src/cypress/cy.coffee

```diff
@@ -727,6 +731,7 @@ create = (specWindow, Cypress, Cookies, state, config, log) ->
         window: s.window
         document: s.document
         $autIframe: s.$autIframe
+        actAsIfWindowHasFocus: s.actAsIfWindowHasFocus
```

Comment:
this appears unnecessary

Reply:
without it, `state('actAsIfWindowHasFocus')` was always false for me